docs/source/using_the_ve/getting_started.md

```diff
@@ -56,18 +56,36 @@ configuration and data files to run a model.
 ve_run --install-example /path/
 ```
 
-You can then run the model itself:
+You can then run the model itself. If you have already run the simulation you will need
+to delete or rename the output files, as previously generated output can prevent the
+simulation from running.
 
 ```shell
 ve_run /path/ve_example/config \
     --outpath /path/ve_example/config/out \
     --logfile /path/ve_example/out/ve_example.log
 ```
 
-The [Virtual Ecosystem in use](virtual_ecosystem_in_use.md) page provides a walkthrough
-of this process, showing the typical outputs of the model run process, and also provides
-some simple plots of model inputs and ouputs.
++++
 
-Once you want to start digging into the structure of the model and inputs, the [example
-data](./example_data.md) pages provides a detailed description of the  contents of the
-`ve_example` directory.
+## Simulation results
+
+The Virtual Ecosystem writes out a number of data files:
+
+* `initial_state.nc`: A single compiled file of the initial input data.
+* `all_continuous_data.nc`: An optional record of time series data of the variables
+  updated at each time step.
+* `final_state.nc`: The model data state at the end of the final step.
+
+These files are written to the standard NetCDF data file format.
```

Comment:
Maybe insert example code to remove them here. I think this is what @jacobcook1995 did in another branch.